src/clustering/administration/main/command_line.cc

```diff
@@ -891,7 +891,8 @@ options::help_section_t get_server_options(std::vector<options::option_t> *optio
                                              options::OPTIONAL));
     help.add("-n [ --server-name ] arg",
              "the name for this server (as will appear in the metadata).  If not"
-             " specified, it will be randomly chosen from a short list of names.");
+			 " specified, one will be generated from the hostname and a random "
```

Comment:
One last thing: Could you make the indentation match the one of the line above ("the name for this ...")? We usually don't add additional indentation for strings that go over multiple lines.